### DISCUSSION TITLE
Apply ISO 3166-2:JP, Japan Prefecture Order on 'anywhere' pull-down menu

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1208] [Ideas]
'anywhere' pull-down menu is currently displayed alphabetic order but if we can use ISO 3166-2:JP, Japan Prefecture Order, it will be good for website in A.A. Japan so please let us know if you have any ideas to use ISO 3166-2:JP for 'anywhere' pull-down menu.

**Comment by joshreisner** [comment]
hi @Hiro-Tokyo what is the URL of the meeting finder? also, which region should come first?

**Reply by joshreisner** [reply]
assuming it's 北海道

**Reply by joshreisner** [reply]
i see there is actually a problem in the sorting - it should be using `localeCompare` for better sorting in localized (non-English) languages, pushing that update now

**Reply by joshreisner** [reply]
give it a refresh and let us know if that helps - this will affect types as well

if not, you may need to prefix the regions with a number or other sortable character. for example

1. manhattan
2. brooklyn
3. bronx
4. queens
5. staten island

**Reply by Hiro-Tokyo** [reply]
Thanks Josh! Seems not affect to Japan prefectures order yet but I see about prefix. 

https://gist.github.com/komasaru/9303050#file-iso_3166_2_jp-csv-L1
![1280px-Regions_and_Prefectures_of_Japan svg](https://github.com/code4recovery/12-step-meeting-list/assets/100449629/1d860641-804d-4f21-bfc8-47ea90b4a11e)
